src/redux/actions/genes/loadMarkerGenes.js

```diff
@@ -6,26 +6,16 @@ import { fetchWork } from '../../../utils/work/fetchWork';
 import getTimeoutForWorkerTask from '../../../utils/getTimeoutForWorkerTask';
 
 const loadMarkerGenes = (
-  experimentId, resolution, plotUuid, numGenes = 5, selectedCellSets = false,
+  experimentId, resolution, plotUuid, numGenes = 5, selectedCellSets,
```

Comment:
why do you still keep this `selectedCellSets` parameter?

Reply:
This is used by the worker to select the `cellSet` that will be used to generate the marker heatmap.

Reply:
Yes, makes sense. I think it will be clearer if you rename it and all its references to something like `selectedClusteringType` and also add `'louvain'` as its default value at the function declaration.

Reply:
It's the `cellSet` that's selected, so we can use `scratchpad` as well, not only limited to the clustering method. I think this name is already suitable

Reply:
Oh, I see

Reply:
let's keep it like this then